main.py

```diff
@@ -60,7 +60,7 @@
     # Check if the message author has the Game Master role
     isGameMaster = False
     if message.guild == gamelog_channel.guild:
-        if game_master in [y.id for y in message.author.roles]:
+        if game_master in [y.id for y in message.author.guild.get_member(message.author.id).roles]:
```

Comment:
Whoops! This needs to change!

Reply:
So does line 264 in interpretation\ww_head.py, but I can't seem to comment on that.